docs/INSTALL_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+file and install telegraf:
+
+```shell
+# influxdata-archive_compat.key GPG fingerprint:
+#     9D53 9D90 D332 8DC7 D6C8 D3B9 D8FF 8E1F 7DF8 B07E
+cat <<EOF | sudo tee /etc/yum.repos.d/influxdata.repo
+[influxdata]
+name = InfluxData Repository - Stable
+baseurl = https://repos.influxdata.com/stable/\$basearch/main
+enabled = 1
+gpgcheck = 1
+gpgkey = https://repos.influxdata.com/influxdata-archive_compat.key
+EOF
+sudo yum install telegraf
+```
+
+## Official Docker images
+
+Telegraf is available as an [Official image][] on DockerHub. Official images
+are a curated set of Docker Images that also automatically get security updates
+from Docker, follow a set of best practices, and are available via a shortcut
+syntax which omits the organization.
+
+InfluxData maintains a Debian and Alpine based image across the last three
+minor releases. To pull the latest Telegraf images:
+
+```shell
+# latest Debian-based image
+docker pull telegraf
+# latest Alpine-based image
+docker pull telegraf:alpine
+```
+
+See the [Telegraf DockerHub][] page for complete details on available images,
+versions, and tags.
+
+[official image]: https://docs.docker.com/trusted-content/official-images/
+[Telegraf DockerHub]: https://hub.docker.com/_/telegraf
+
+## Helm charts
+
+A community-supported [helm chart][] is also available:
+
+```shell
+helm repo add influxdata https://helm.influxdata.com/
+helm search repo influxdata
+```
+
+[helm chart]: https://github.com/influxdata/helm-charts/tree/master/charts/telegraf
+
+## Nightly builds
+
+[Nightly builds][] are available and are generated from the master branch each
+day at around midnight UTC. The artifacts include both binary packages, RPM &
+DEB packages, as well as nightly Docker images that are hosted on [quay.io][].
+
+[Nightly builds]: /docs/NIGHTLIES.md
+[quay.io]: https://quay.io/repository/influxdb/telegraf-nightly?tab=tags&tag=latest
+
+## Build from source
+
+Telegraf generally follows the latest version of Go and requires GNU make to use
+the Makefile for builds.
+
+On Windows, the makefile requires the use of a bash terminal to support all
+makefile targets. An easy option to get bash for windows is using the version
+that comes with [git for windows](https://gitforwindows.org/).
+
+1. [Install Go](https://golang.org/doc/install)
+2. Clone the Telegraf repository:
+
+   ```shell
+   git clone https://github.com/influxdata/telegraf.git
+   ```
+
+3. Run `make build` from the source directory
+
+   ```shell
+   cd telegraf
+   make build
+   ```
+
+## Custom builder
+
+Telegraf also provides a way of building a custom minimized binary using the
+[custom builder][]. This takes a user's configuration file(s), determines what
+plugins are required, and builds a binary with only those plugins. This greatly
+reduces the size of the Telegraf binary.
+
+[custom builder]: https://github.com/influxdata/telegraf/tree/master/tools/custom_builder
```

Comment:
Can we please use a relative path here? In case we ever want to create some HTML pages or similar from the markdown files it would be helpful to distinguish between "internal" links and external ones...

Reply:
Good catch, I've pushed an update and verified it works as well.